src/kfactory/kcell.py

```diff
@@ -5886,6 +5822,11 @@ def dcenter(self) -> kdb.DPoint:
         """Coordinate of the port in um."""
         return self.bbox().center()
 
+    def insert(self, cell: KCell | VKCell) -> VInstance:
+        vi = VInstance(cell)
+        self.insts.append(vi)
+        return vi
```

Comment:
this would be the same as `insert_vinst`, correct? If so, just delegate to that one instead of duplicating code.